### ISSUE TITLE
Bottom navigation with named routing?

### ISSUE THREAD

**Issue by Reprevise** [GIfatahTH/states_rebuilder#262]
Is it possible with `states_rebuilder` to have a bottom navigation bar with named routing? The bottom navigation bar would also need its own navigation stack per tab.

**Comment by amoslai5128** [comment]
> bottom navigation

Yes, I think it does the same logic with drawer, the bottom navigation bar is a UI widget. 

Can you provide your code as an example?

**Comment by Reprevise** [comment]
> > bottom navigation
> 
> Yes, I think it does the same logic with drawer, the bottom navigation bar is a UI widget.
> 
> Can you provide your code as an example?

I don't have any code, and I'm aware of the `BottomNavigationBar` widget. I'm just wondering if its possible to do a bottom navigation bar with named routing with an individual navigation stack per bottom bar tab.
